Data/Database.cs

```
@@ -10,14 +10,52 @@ namespace MauiApp1.Data
     public class Database
     {
         public SQLiteAsyncConnection _database;
+        public Database()
+        {
+            InitializeDatabaseAsync().Wait(); // Ensure initialization is complete before proceeding
```

Use .Result instead of Wait(), this swallows the exception if InitializeDatabaseAsync throws one.